src/twisted/trial/test/test_assertions.py

```diff
@@ -1706,14 +1713,6 @@ def somethingOld():
 _somethingOld = somethingOld
 
 
-deprecatedModuleAttribute(
```

Comment:
maybe is better to have it here, to be closer to the deprecation target.

Or maybe move it just before `def somethingOld'

Reply:
It's not clear to me why this needs to be touched at all.

Reply:
This emits a warning when the tests run. That said, it's a fake used for testing that deprecated code emits warnings. I agree that this probably isn't in scope of "don't import code deprecated in twisted or in the python stdlib." Reverting.